core/arch/arm/plat-ti/main.c

```diff
@@ -175,3 +176,24 @@ void console_init(void)
 			     CONSOLE_UART_CLK_IN_HZ, CONSOLE_BAUDRATE);
 	register_serial_console(&console_data.chip);
 }
+
+#if defined(CFG_OTP_SUPPORT)
+
+void tee_otp_get_hw_unique_key(struct tee_hw_unique_key *hwkey)
+{
+	memcpy(&hwkey->data[0], &plat_huk[0], sizeof(hwkey->data));
+}
+
+/* TODO */
```

Comment:
Indeed. I'll pick up the first three patches from https://github.com/OP-TEE/optee_os/pull/1318.